src/views/studio/CreateMemberView/CreateMemberView.tsx

```diff
@@ -54,6 +56,13 @@ const CreateMemberView = () => {
     throw queryNodeStateError
   }
 
+  const [getMember, { data: memberData }] = useLazyQuery<GetMembershipQuery, GetMembershipQueryVariables>(
```

Comment:
Had some struggles with introducing custom lazy hook. It would require some additional work. Since it will be used only here I decided to stick with it. If we will need more lazy hooks (or we will use that one frequently) in the future I believe we can add them then.

Reply:
No problem, I'm okay with that 👍🏻

Reply:
Our codegen actually generates the lazy hooks as well. If we don't want to go through `@/api/hooks`, let's just use `useGetMembershipLazyQuery` that's already generated